GuildedRoseLLC/Items/ItemsViewController.swift

```diff
@@ -44,6 +43,19 @@ public class ItemsViewController: UIViewController, UICollectionViewDelegate {
         
         itemCollectionView.dataSource = dataSource
         itemCollectionView.delegate = self
-        
+    }
+    
+    public override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "DetailsPageSegue" {
```

Comment:
Would it be possible to extract this string into a constant? Right now, `"DetailsPageSegue"` is a magic string that is duplicated in the production code and test.

Reply:
To clarify, I think the production code and test should reference the same constant.

Reply:
extracted